dashboard/src/shared/Chart.ts

```diff
@@ -23,5 +24,13 @@ export default class Chart {
     return true;
   }
 
+  public static async checkUpdates(name: string, version: string, appVersion: string) {
```

Comment:
I think this method should match with the endpoint/handler on the API side. So I think this method could be called `listChartsWithFilters`.